json-path/src/main/java/com/jayway/jsonpath/internal/function/sequence/Distinct.java

```diff
@@ -0,0 +1,31 @@
+package com.jayway.jsonpath.internal.function.sequence;
+
+import com.jayway.jsonpath.JsonPathException;
+import com.jayway.jsonpath.internal.EvaluationContext;
+import com.jayway.jsonpath.internal.PathRef;
+import com.jayway.jsonpath.internal.function.Parameter;
+import com.jayway.jsonpath.internal.function.PathFunction;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Take the unique items from collection of JSONArray
+ *
+ * Created by PavelSakharchuk on 21/09/23
+ */
+public class Distinct implements PathFunction {
+
+    @Override
+    public Object invoke(String currentPath, PathRef parent, Object model, EvaluationContext ctx, List<Parameter> parameters) {
```

Comment:
please make sure this function works with arrays of objects as well

Reply:
Thanks. Added tests on the array of objects and empty array.